Submitted By: Mario Fetka (geos_one) (mario dot fetka at gmail dot com)
Date: 2010-04-04
Initial Package Version: 0.16
Origin: me
Upstream Status: unknown
Description: make the testsuite run with the new mltilib design
 
diff -Naur a/chrpath-0.16/testsuite/runtest.sh b/chrpath-0.16/testsuite/runtest.sh
--- a/testsuite/runtest.sh	2010-04-04 09:02:21.859114159 +0000
+++ b/testsuite/runtest.sh	2010-04-04 09:42:08.762112560 +0000
@@ -2,6 +2,8 @@
 
 set -x
 
+export LD_LIBRARY_PATH=$PWD/../.libs:$LD_LIBRARY_PATH
+
 CHRPATH=../chrpath
 
 retval=0
